deploy.sh

```diff
@@ -24,7 +24,7 @@ STACK=${SUB_DOMAIN}
 
 PROVISIONED_CONCURRENT_EXECUTIONS=${PROVISIONED_CONCURRENT_EXECUTIONS-'1'}
 RESERVED_CONCURRENT_EXECUTIONS=${RESERVED_CONCURRENT_EXECUTIONS-'3'}
-
+cat template.yml.erb | ruby -rerb -e "puts ERB.new(ARGF.read, nil, '-').result" > template.yml
```

Comment:
I think we can use the command line `erb` utility that's included with Ruby and set the trim mode to `-`?

``` bash
erb -T - template.yml.erb > template.yml
```

Reply:
👍